config/crd/bases/osp-director.openstack.org_openstackprovisionservers.yaml

```diff
@@ -51,7 +51,7 @@ spec:
               OpenStackProvisionServer
             properties:
               apacheImageUrl:
-                default: registry.redhat.io/rhel8/httpd-24:1-152
+                default: registry.redhat.io/rhel8/httpd-24:latest
```

Comment:
when using latest, should we set `imagePullPolicy: Always` in the controller creating the osp-httpd container?

Reply:
yes! Just added a new commit to address this